src/module/actor/sheet/helpers.ts

```diff
@@ -80,4 +80,27 @@ function createAbilityViewData(item: AbilityItemPF2e | FeatPF2e): AbilityViewDat
     };
 }
 
-export { condenseSenses, createAbilityViewData, createBulkPerLabel, onClickCreateSpell };
+/**
+ * Applies a delta change to the value of an input element
+ *
+ * @param input reference to the input field
+ * @param delta the amount of change to apply to the input's current value
+ * @returns the new value of the input field
+ */
+function applyDeltaToInput(input: HTMLInputElement, delta: number): string {
+    const min = Number(input.dataset.min) || 0;
+    const max = Number(input.dataset.max) || 0;
+    const oldValue = Number(input.value) || min;
+
+    if (max > 0) {
+        input.value = String(Math.clamp(oldValue + delta, min, max));
+    } else {
+        input.value = String(Math.max(oldValue + delta, min));
+    }
+
+    fu.debounce((el: HTMLInputElement) => el.dispatchEvent(new Event("change", { bubbles: true })), 0)(input);
+
+    return input.value;
```

Comment:
Doesn't seem like you use the return value. You can omit it I think.

Reply:
That's true, cleaned up!